Tempo/AppDelegate.swift

```diff
@@ -147,7 +160,108 @@ class AppDelegate: UIResponder, UIApplicationDelegate, SWRevealViewControllerDel
+		
+		let fbBodyString = "Tempo is a music sharing application that allows you to share 30 second clips with your friends."
+		let paragraphStyle = NSMutableParagraphStyle()
+		let attrString = NSMutableAttributedString(string: fbBodyString)
+		paragraphStyle.lineSpacing = 12
+		attrString.addAttribute(NSParagraphStyleAttributeName, value:paragraphStyle, range:NSMakeRange(0, attrString.length))
+		
+		let fbOnboard = OnboardingContentViewController(title: "", body: fbBodyString, image: UIImage(named: "logo"), buttonText: "Connect with Facebook") { () -> Void in
+			self.loginToFacebook()
+		}
+		
+		fbOnboard.topPadding = 160
+		fbOnboard.underIconPadding = 40;
+		
+		fbOnboard.bodyLabel.attributedText = attrString
+		fbOnboard.bodyLabel.textAlignment = NSTextAlignment.Center
+		fbOnboard.bodyLabel.textColor = fadedGrey
+		fbOnboard.bodyLabel.font = UIFont(name: "Avenir Next Regular", size: 17)
+		fbOnboard.bodyLabel.font = fbOnboard.bodyLabel.font.fontWithSize(17)
+		
+		fbOnboard.actionButton.layer.cornerRadius = 5
+		fbOnboard.actionButton.backgroundColor = fbBlue
+		fbOnboard.actionButton.titleLabel!.font =  UIFont(name: "Avenir Next Regular", size: 17)
+		fbOnboard.actionButton.titleLabel!.font = fbOnboard.actionButton.titleLabel!.font.fontWithSize(17)
+		
+		return fbOnboard
+	}
+	
+	func generateSpotifyOnboardVC() -> OnboardingContentViewController {
+		let spotifyGreen = UIColor(red: 5.0/255.0, green: 153.0/255.0, blue: 82.0/255.0, alpha: 1)
+		
+		let spotifyOnboard = OnboardingContentViewController(title: "", body: "Add the songs you like to your Spotify library.", image: UIImage(named: "spotify-connect"), buttonText: "Connect to Spotify ") { () -> Void in
+			SpotifyController.sharedController.loginToSpotify { (success) in
+				if success {
+					if let session = SPTAuth.defaultInstance().session {
+						if session.isValid() {
+							SpotifyController.sharedController.setSpotifyUser(session.accessToken)
+							self.toggleRootVC()
+						}
+					}
+				}
+			}
+		}
 		
+		spotifyOnboard.topPadding = 200
+		spotifyOnboard.bottomPadding = 60
+		
+		let title = UILabel(frame: CGRectMake(37, 102, 316, 28))
+		title.textAlignment = NSTextAlignment.Center
+		title.text = "Connect to Spotify"
+		title.textColor = fadedGrey
+		title.font = UIFont(name: "Avenir Next Regular", size: 29)
+		title.font = title.font.fontWithSize(29)
+		spotifyOnboard.view.addSubview(title)
+		
+		spotifyOnboard.bodyLabel.textColor = darkGrey
+		spotifyOnboard.bodyLabel.font = UIFont(name: "Avenir Next Regular", size: 20)
+		spotifyOnboard.bodyLabel.font = spotifyOnboard.bodyLabel.font.fontWithSize(20)
+		
+		spotifyOnboard.actionButton.layer.cornerRadius = 5
+		spotifyOnboard.actionButton.backgroundColor = spotifyGreen
+		spotifyOnboard.actionButton.titleLabel!.font =  UIFont(name: "Avenir Next Regular", size: 17)
+		spotifyOnboard.actionButton.titleLabel!.font = spotifyOnboard.actionButton.titleLabel!.font.fontWithSize(17)
```

Comment:
you should be using a ? instead of an ! here

spotifyOnboard.actionButton.titleLabel?.font = ...